pta-intelligent-search-metadata-extractor/src/main/java/fi/maanmittauslaitos/pta/search/metadata/model/CodeListValue.java

```diff
@@ -0,0 +1,76 @@
+package fi.maanmittauslaitos.pta.search.metadata.model;
+
+import java.util.Objects;
+
+public class CodeListValue {
+
+    // TODO: PAIK-1949 re-evaluate these location
```

Comment:
Remove comment that refers to other trackers

Reply:
Removed